cli/app/commands/conflict/command.py

```diff
@@ -0,0 +1,55 @@
+import typer
+from .conflict import ConflictConfig, ConflictService
+from .messages import (
+    conflict_check_help,
+    error_checking_conflicts,
+    conflicts_found_warning,
+    no_conflicts_info,
+    checking_conflicts_info,
+)
+from app.utils.logger import Logger
+
+conflict_app = typer.Typer(help=conflict_check_help, no_args_is_help=False)
+
+
+@conflict_app.callback(invoke_without_command=True)
+def conflict_callback(
+    ctx: typer.Context,
+    config_file: str = typer.Option("helpers/config.prod.yaml", "--config-file", "-c", help="Path to configuration file"),
+    timeout: int = typer.Option(5, "--timeout", "-t", help="Timeout for tool checks in seconds"),
+    verbose: bool = typer.Option(False, "--verbose", "-v", help="Verbose output"),
+    output: str = typer.Option("text", "--output", "-o", help="Output format (text/json)"),
+) -> None:
+    """Check for tool version conflicts"""
+    if ctx.invoked_subcommand is None:
+        # Initialize logger once and reuse throughout
+        logger = Logger(verbose=verbose)
+        
+        try:
+            logger.info(checking_conflicts_info)
+
+            config = ConflictConfig(
```

Comment:
make use of the timeout wrapper here like other commands instead of passing the timeout variable